etddf_minau/Measurement[] measurements
etddf_minau/Measurement[] all_measurements
string src_asset
float32 delta_multiplier